src/store/creator/selectors.ts

```diff
@@ -1,12 +1,11 @@
 import { AppState } from '../types';
 import { CreatorItem, CreatorItemHighlight, CreatorItemRegion, CreatorStatus } from './types';
-import { Shape } from '../../@types';
```

Comment:
Can we delete this Shape type or is it still used elsewhere?

Reply:
I think it's still used elsewhere